### PR TITLE
initial travis

### PR DESCRIPTION
**Pull request by brianshannan-wf**
@Workiva/messaging-pp

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

**Comment by stevenosborne-wf** [comment]
+1

**Comment by charliestrawn-wf** [comment]
lgtm

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.x
```
Comment (aldenpeterson-wf) [inline .travis.yml]: You might want to explicitly set 1.8 here or a go update will blow up the build

**Comment by jacobmoss-wf** [comment]
+1

**Comment by stevenosborne-wf** [comment]
+1

**Pull request merged**